docs/en/component_specs/sequencer/snowflake.md

```diff
@@ -0,0 +1,65 @@
+# Snowflake
```

Comment:
Please add this doc into the english sidebar `docs/_sidebar.md`

Reply:
got it